### ISSUE TITLE
Connection over SSH failing with El Capitan

### ISSUE THREAD

**Issue by ravi** [sequelpro/sequelpro#2224]
I am guessing SSH was updated in El Capitan and SequelPro seems to be using a port/host/port syntax for the SSH tunnel that may no longer be supported:
```
Used command:  /usr/bin/ssh -v -N -o ControlMaster=no -o ExitOnForwardFailure=yes -o ConnectTimeout=10 -o NumberOfPasswordPrompts=3 -o TCPKeepAlive=no -o ServerAliveInterval=60 -o ServerAliveCountMax=1 xyz@........ -L 55646/127.0.0.1/3306

Bad local forwarding specification ' 55646/127.0.0.1/3306'
```
Changing that to ":"s might fix the issue.

**Comment by cokeperez** [comment]
+1.

Same problem here.

**Comment by dmoagx** [comment]
Duplicate of #2050 
This has already been fixed since 1.1rc1: http://www.sequelpro.com/blog/2015.08/sequel-pro-1-1-rc2/
Please search for existing issues, before creating a new one.

**Comment by ravi** [comment]
Did search actually, except it looks like GitHub does not search closed issues and #2050 is closed. Learnt a lesson for the next time. Thank you for the fix.
